### PR TITLE
🩹 made app draggable

### PR DESCRIPTION
**Pull request by alvinlal**
fixed issue #2

### CODE REVIEW

File: main/src/app/pages/file-manager/file-manager.component.css [hunk 1]
```diff
@@ -80,6 +83,7 @@
   margin: 0px 5px;
   transform: scale(0.8);
   transition: transform 0.3s;
+  -webkit-app-region: no-drag;
```
Comment (alvinlal) [inline main/src/app/pages/file-manager/file-manager.component.css:86]: enabling drag on buttons would make them non clickable

File: main/src/app/pages/file-manager/file-manager.component.css [hunk 2]
```diff
@@ -66,6 +66,9 @@
   align-items: center;
   overflow-x: auto;
   background: var(--primary);
+  -webkit-app-region: drag;
```
Comment (alvinlal) [inline main/src/app/pages/file-manager/file-manager.component.css:69]: enable drag using -webkit-app-region property

File: main/src/app/pages/file-manager/file-manager.component.css [hunk 3]
```diff
@@ -66,6 +66,9 @@
...
+  -webkit-user-select: none;
+  user-select: none;
```
Comment (alvinlal) [inline main/src/app/pages/file-manager/file-manager.component.css:71]: also define the standard property user-select for compatibility

**Comment by 27px** [comment]
Thanks @alvinlal

**Pull request merged**